db/migrate/20170509202157_change_uniqueness_index_on_topic_status_updates.rb

```diff
@@ -0,0 +1,21 @@
+class ChangeUniquenessIndexOnTopicStatusUpdates < ActiveRecord::Migration
+  def up
+    execute "DROP INDEX idx_topic_id_status_type_deleted_at"
+
+    execute <<~SQL
+    CREATE UNIQUE INDEX idx_topic_id_status_type_user_id_deleted_at
```

Comment:
I don't think this is what we want here. This index means that other topic status updates like `close topic` can be set more than once as long as it is created by a different user. Perhaps a json column that can store dynamic data might be better here? In the column we can maintain the list of user ids and the timings that they would want to be reminded at.